Source/geom.c

```diff
@@ -259,3 +260,22 @@ void tesedgeIntersect( TESSvertex *o1, TESSvertex *d1,
 		v->t = Interpolate( z1, o2->t, z2, d2->t );
 	}
 }
+
+/*
+	Calculate the angle between v1-v2 and v1-v0
+ */
+TESSreal calcAngle(TESSvertex *v0, TESSvertex *v1, TESSvertex *v2) {

```

Comment:
Make function signature consistent with rest of the code. I'm not big fan of the style (picked it up from the original libtess), but let's be consistent:
```
TESSreal calcAngle( TESSvertex *v0, TESSvertex *v1, TESSvertex *v2 )
{
```